deploy/csi-nfs-driverinfo.yaml

```diff
@@ -7,4 +7,5 @@ spec:
   attachRequired: false
   volumeLifecycleModes:
     - Persistent
+    - Ephemeral
```

Comment:
pls add this config into https://github.com/kubernetes-csi/csi-driver-nfs/blob/master/charts/latest/csi-driver-nfs/templates/csi-nfs-driverinfo.yaml, and then update `tgz` file by `helm package charts/latest/csi-driver-nfs -d charts/latest`, thanks.